### DISCUSSION TITLE
Is there any shared subscription limit?

### DISCUSSION THREAD

**Discussion by 1995parham** [emqx/emqx#6300] [Q&A] [answered]
Hi :wave: 

We have set up a cluster with 25 nodes of EMQ in our company Kubernetes. We have internal and external zones and our internal zone has a shared subscription. I can see on the EMQ dashboard that the shared subscription maximum is 4 but I cannot see any configuration on this. can you please help me with that?

**Comment by thalesmg** [comment, marked as answer]
Hi @1995parham !

Do you mean the shared subscription maximum as highlighted in the following screenshot?

![image](https://user-images.githubusercontent.com/16166434/143877219-d9b37d8a-7d2e-4996-b608-11dd15e176ca.png)

If so, this number only represents the maximum number of shared subscriptions observed during the lifetime of those statistics. It is not controlled by any configurations, but only measured.

**Reply by 1995parham** [reply]
Thanks for your awesome explanation.
